js/lib/comments.js

```diff
@@ -113,6 +113,10 @@ elgg.comments.init = function() {
 		dc.toggleEdit();
 		return false;
 	});
+
+	// Highlight a comment if its id is found from the URL
+	var commentId = elgg.getSelectorFromUrlFragment(document.URL);
+	$(commentId).addClass('elgg-highlight').delay(1000).removeClass('elgg-highlight', {duration: 1000});
```

Comment:
At first I though it wasn't even possible, but it ended up being a regular fade-away animation.